Makefile

```diff
@@ -97,23 +99,35 @@ $(I): $(O)
 .PHONY: sdcard
 sdcard: u_boot $(I) $(D)
 # Create image with partition table
-	$(Q)dd if=/dev/zero of=$(D)/sdcard.img bs=1M count=36
+	$(Q)dd if=/dev/zero of=$(D)/sdcard.img bs=1M count=18
 	$(Q)parted --script $(D)/sdcard.img \
 		mklabel msdos \
-		mkpart primary fat16 4MiB 100% \
+		mkpart primary fat16 2MiB 10MiB \
```

Comment:
Why is the boot partition's offset changed from 4MB to 2MB? Some SD cards will have issues with that (erase boundary is 4MB for some flash types). More important, some older ROMs (like AM65 SR1.0 IIRC) can also have issues.

Reply:
I was trying to minimize the image size to improve download performance.  I didn't realize 2MB offset isn't supported on some SDcards.  Seems like that is a bug with the sdcard??  But anyway, I can change back.

Reply:
It should be "supported" on all SD cards, but you may take a performance hit on some. My real concern is with ROM support.